enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/TopicEntityImpl.java

```diff
@@ -0,0 +1,126 @@
+package eu.europeana.enrichment.model.impl;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import eu.europeana.enrichment.model.TermEntity;
+import eu.europeana.enrichment.model.TopicEntity;
+import eu.europeana.enrichment.model.TopicModel;
+
+public class TopicEntityImpl implements TopicEntity {
+	
+	private String id;
```

Comment:
please rename to topicId and and the mongo database id and the required morphia annotations like in other classes: https://github.com/EnrichEuropeana/enrichment/blob/188681b29507b0f925af728c6418449e75a282b3/enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/ItemEntityImpl.java